app/models/setting/provisioning.rb

```diff
@@ -29,10 +29,10 @@ def self.local_boot_labels
     'lo',
     'en*v*',
     'usb*',
-    'vnet*',
+    '*vnet*',
     'macvtap*',
     ';vdsmdummy;',
-    'veth*',
+    '*veth*',
```

Comment:
This is a virtual ethernet and the naming convention is **typically** just `vethN`: https://github.com/torvalds/linux/blob/f40ddce8/drivers/net/veth.c#L30

I think it is safe to keep just `veth*` here.